cleanlab_studio/studio/studio.py

```diff
@@ -164,7 +163,7 @@ def create_project(
         project_name: str,
         modality: Literal["text", "tabular", "image"],
         *,
-        task_type: Literal["multi-class", "multi-label", "regression"] = "multi-class",
+        task_type: Optional[Literal["multi-class", "multi-label", "regression"]] = "multi-class",
```

Comment:
should task_type be allowed to take value "unsupervised"?

Reply:
can we add description about how to use the tasktype arg in docstring

Reply:
Was planning to tackle docs updates in a separate PR since docstring updates here will be immediately reflected at help.cleanlab.ai when a new python API version is released, so that should be the last step before releasing the feature. I have a [subtask](https://app.asana.com/0/0/1206290585748540/f) in asana to track this